src/ai/backend/manager/registry.py

```diff
@@ -2446,13 +2467,20 @@ async def _decrease_concurrency_used(access_key: AccessKey, is_private: bool) ->
                                     reason=reason,
                                     status_changed_at=now,
                                 )
-                                await self.event_producer.produce_event(
+                                await self.event_producer.anycast_event(
                                     SessionCancelledEvent(
                                         session_id,
                                         target_session.creation_id,
                                         reason,
                                     ),
                                 )
+                                await self.event_producer.broadcast_event(
+                                    SessionCancelledBroadcastEvent(
+                                        session_id,
+                                        target_session.creation_id,
+                                        reason,
+                                    ),
+                                )
```

Comment:
I think there was a function called `anycast_and_broadcast_event`, shouldn't we use that function?
It's not that the above function is good, but I want to easily check which parts need to be modified later.

It would be good to unify all parts where anycast and broadcast are used together with the above function, and then modify all parts using the above function in a batch later.